test/fixtures/rules/std_rule_cases/L029.yml

```diff
@@ -114,3 +114,25 @@ test_fail_keyword_as_quoted_identifier_column_alias_config:
         quoted_identifiers_policy: column_aliases
     core:
       dialect: tsql
+
+test_pass_ignore_word1:
+  fail_str: CREATE TABLE artist(create TEXT)
+  configs:
+    rules:
+      L029:
+        ignore_word: create
+
+
+test_pass_ignore_word2:
+  pass_str: SELECT col1 AS date FROM table1
+  configs:
+    rules:
+      L029:
+        ignore_word: date
+
+
+test_pass_one_character_identifier:
+  pass_str: SELECT d.col1 FROM table1 d
+  configs:
+    core:
+      dialect: Snowflake
```

Comment:
Nitpick: Did you mean to capitalize the "S" in "snowflake"? It's the first time I've seen capital letters when referring to dialects.

Reply:
No - that's a good spot! It doesn't work with that, hence tests failing.